modules/doc/content/newsletter/2022_01.md

```diff
@@ -32,6 +32,16 @@ moose-petsc              3.15.1             build_4
 
 ## libMesh-level Changes
 
+- Check processor id of point neighbors
+- Nanoflann update to 1.4.1
+- Update to TIMPI 1.8.1
+- Use get_unique_tag() for a few more MessageTags
+- Remove caching from FE<CLOUGH>
+- Higher precision when printing assert failures
+- resolve warning of expectedly unbinned entries.
+- Clear input data structure before using
+- Don't unnecessarily compute the Jacobian for ComputeResidualandJacobian
+
```

Comment:
We should probably filter out the less interesting entries here (nobody cares about some binning warning) and expand the more interesting ones.

Reply:
I find binning warnings fascinating 🤣

Reply:
I'm honestly fine with having everything in there - these newsletters should be useful to everybody, regardless of their opinion on binning warnings 😉